ocaml/xapi/pool_db_backup.ml

```diff
@@ -34,7 +34,7 @@ let write_database (s: Unix.file_descr) ~__context =
     (* CA-18377: If we're in the middle of a rolling upgrade from Miami *)
     (* to Orlando, then only send a minimally-compliant Miami database. *)
     (* Orlando hosts will ignore this database and carry on.            *)
-    let len = String.length minimally_compliant_miami_database in
+    let len = Bytes.length minimally_compliant_miami_database in
     ignore (Unix.write s minimally_compliant_miami_database 0 len)
```

Comment:
This could use `Unix.write_substring` and then you may not have to make `minimally_compliant_miami_database` mutable.

Reply:
Done!